mne/minimum_norm/time_frequency.py

```diff
@@ -447,94 +452,125 @@ def compute_source_psd(raw, inverse_operator, lambda2=1. / 9., method="dSPM",
-    freqs = freqs[freqs_mask]
-    fstep = np.mean(np.diff(freqs))
-    if is_free_ori and pick_ori is None:
-        psd = np.zeros((K.shape[0] // 3, np.sum(freqs_mask)))
-    else:
-        psd = np.zeros((K.shape[0], np.sum(freqs_mask)))
-    n_windows = 0
+        .. versionadded:: 0.17
+    low_bias : bool
+        Only use tapers with more than 90% spectral concentration within
+        bandwidth.
 
-    for this_start in np.arange(start, stop, int(n_fft * (1. - overlap))):
-        data, _ = raw[sel, this_start:this_start + n_fft]
-        if data.shape[1] < n_fft:
-            logger.info("Skipping last buffer")
-            break
+        .. versionadded:: 0.17
+    n_jobs : int
+        Number of parallel jobs to use (only used if adaptive=True).
 
-        if Vh is not None:
-            data = np.dot(Vh, data)  # reducing data rank
+        .. versionadded:: 0.17
+    return_sensor : bool
+        If True, return the sensor PSDs as an EvokedArray.
 
-        data *= window[None, :]
+        .. versionadded:: 0.17
+    dB : bool
+        If True (default in 0.17, will change to False in 0.18),
+        return output it decibels.
 
-        data_fft = fftpack.fft(data)[:, freqs_mask]
-        sol = np.dot(K, data_fft)
+        .. versionadded:: 0.17
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see :func:`mne.verbose`
+        and :ref:`Logging documentation <tut_logging>` for more).
 
-        if is_free_ori and pick_ori is None:
-            sol = combine_xyz(sol, square=True)
-        else:
-            sol = (sol * sol.conj()).real
+    Returns
+    -------
+    stc : SourceEstimate | VolSourceEstimate
+        The PSD (in dB) of each of the sources.
 
-        if method != "MNE":
-            sol *= noise_norm ** 2
+    See Also
+    --------
+    compute_source_psd_epochs
 
-        psd += sol
-        n_windows += 1
+    Notes
+    -----
+    Each window is multiplied by a window before processing, so
+    using a non-zero overlap is recommended.
 
-    psd /= n_windows
+    This function is different from :func:`compute_source_psd_epochs` in that:
 
-    psd = 10 * np.log10(psd)
+    1. ``dB=True`` by default (deprecated; will change to False in 0.18)
+    2. ``bandwidth='hann'`` by default, skipping multitaper estimation
+    3. For convenience it wraps
+       :func:`mne.make_fixed_length_events` and :class:`mne.Epochs`.
 
-    subject = _subject_from_inverse(inverse_operator)
-    stc = _make_stc(psd, vertices=vertno, tmin=fmin * 1e-3,
-                    tstep=fstep * 1e-3, subject=subject,
-                    src_type=inverse_operator['src'].kind)
-    return stc
+    Otherwise the two should produce identical results.
+    """
+    if dB is None:
+        dB = True
+        warn('dB=True by default in 0.17 but will change to False in 0.18, '
+             'set it explicitly to avoid this warning', DeprecationWarning)
```

Comment:
Fix: Deprecate `dB=True` to unify with `compute_source_psd_epochs` (which does not even have a `dB` argument and outputs non-dB PSDs)